src/richText.js

```diff
@@ -94,13 +97,13 @@ define([
                     container: 'body',
                     placement: 'bottom',
                     title: '<h3>' + util.escape(displayId) + '</h3>' +
-                           '<div class="text-muted">' + util.escape(xpath) + '</div>',
+                           '<div class="text-muted">' + util.escape(widget.mug.form.normalizeHashtag(xpath)) + '</div>',
                     html: true,
                     content: '<p>' + labelText.text() + '</p>',
                     template: '<div contenteditable="false" class="popover rich-text-popover">' +
                         '<div class="popover-inner">' +
                         '<div class="popover-title"></div>' +
-                        '<div class="popover-content"><p></p></div>' +
+                        (isCase ? '' : '<div class="popover-content"><p></p></div>') +
```

Comment:
Would it make sense to invert this expression and use `isForm` (a new variable similar to `isCase`, but true if the path starts with `#form/`)? That way if we add more external reference hashtags similar to `#case/` we won't need to change this code.